static/js/periodShortcuts.js

```diff
@@ -0,0 +1,44 @@
+var _ = require('ep_etherpad-lite/static/js/underscore');
+var utils = require('./utils');
+
+var DOT_KEY_CODE = 190;
+var SCENE_EXTENSIONS = ["EXT.", "INT.", "E/I.", "I/E."];
```

Comment:
isso aqui seria bom a gente pegar do plugin de autocomplete, já que é lá que esses valores são definidos. Até pq se algum dia a gente tiver outros valores em outro idioma, vai quebrar isso aqui e a gente nem vai se lembrar disso...

Se for pegar de lá, isso criaria uma dependência entre o ep_SE e o ep_script_autocomp que não seria legal -- é ok o autocomp saber da existência do ep_SE, mas não o contrário, já que o ep_SE existe sem ter o AC habilitado. Uma solução para isso seria mover esse código do shortcut para o ep_script_element_transitions, e tirar ele do ep_SE.